### PR TITLE
Bump charts to add AuthPolicy

### PR DESCRIPTION
**Pull request by sayanh**
**Description**

Changes proposed in this pull request:

- Bumped charts
- Take out 9090/metrics for queue-proxy from Prometheus(details are [here](https://github.com/knative/serving/issues/4638))

**Related issue(s)**

See #7240

### CODE REVIEW

File: resources/knative-serving/templates/pod-monitor-ksvc.yaml [hunk 1]
```diff
@@ -7,8 +7,6 @@ metadata:
 {{ toYaml .Values.knative_serving.podmonitor.labels | indent 4 }}
 spec:
   podMetricsEndpoints:
-  - port: queue-metrics
-    interval: 3s
```
Comment (k15r) [inline resources/knative-serving/templates/pod-monitor-ksvc.yaml]: how is this related to the image bump?

Comment (sayanh) [reply resources/knative-serving/templates/pod-monitor-ksvc.yaml]: Once https://github.com/kyma-project/kyma/pull/7349/files is merged, prometheus will not be able to scrape the port `queue-metrics`. Additionally, it should not be scraped by Prometheus as per this [comment](https://github.com/knative/serving/issues/4638#issuecomment-509445150).

File: resources/application-connector/values.yaml [hunk 2]
```diff
@@ -20,7 +20,8 @@ global:
     path: eu.gcr.io/kyma-project
   application_operator:
     dir:
-    version: a6ce3a2c
+    version: 39d1ab87
+    strict_mode: disabled
```
Comment (Szymongib) [inline resources/application-connector/values.yaml]: Can we move it to the `application-operator` sub chart?
